packages/card-react/example/index.tsx

```diff
@@ -1,15 +1,87 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import { Card } from "../src";
+import { BulletList, BulletListItem } from "@fremtind/jkl-bullet-list-react";
 
-const Sample = () => (
-    <Card>
-        <h1>Heading</h1>
-        <p>
-            For half-an-hour we were muffled in a cold, damp mist, and total darkness, and had begun to think of going
-            indoors when, all at once, the car burst into the pure and starlit region of the upper air.
-        </p>
-    </Card>
+function handleClick() {
+    console.log("This is working");
+}
+
+const CardDemo = () => (
+    <>
+        <a href="https://fremtind.github.io/jokul/" className="clickable">
+            <Card title="Basic Card">
+                <h2>Clickable</h2>
+                <p>
+                    Først ser vi om vi kan behandle saken din automatisk. Da får du svar samme dag. Hvis den ikke kan
+                    behandles automatisk, vurderer vi saken og kontakter deg innen 8 arbeidsdager.
+                </p>
+            </Card>
+        </a>
+        <br></br>
+
+        <Card
+            title="Kjøp bilforsikring"
+            media={{
+                src:
+                    "https://images.unsplash.com/photo-1527867285503-73b0f9cc791f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1567&q=80",
```

Comment:
Foretrekker at vi henter bildene lokalt, så er det ikke nødendig med internet for å jobbe